cluster/search/client.py

```diff
@@ -124,17 +125,30 @@ def search(self, query, start_date=None, end_date=None, limit=100, only_document
                             }
                         }
                     ],
-                    "filter": {
-                        "range": {
-                            "date": {
-                                "gte": start_date or EPOCH,
-                                "lt": end_date or datetime.datetime.now()
+                    "filter": [
+                        {
+                            "range": {
+                                "date": {
+                                    "gte": start_date or EPOCH,
+                                    "lt": end_date or datetime.datetime.now()
+                                }
                             }
                         }
-                    }
+                    ]
+                },
+            },
+            "highlight" : {
```

Comment:
This actually got left in by accident when testing to see if it did what I thought it did